### DISCUSSION TITLE
development branch

### DISCUSSION THREAD

**Discussion by ryanhtafesa** [ryanhtafesa/T24S2T1EXTG04-REV3#2] [General]
Hi @AlexandraKoshur

I have created the development branch and pushed it to the repo ready for cloning to build your feature.

In the NavigationPage, there is already click events for each button in the XAML ready to go.

When testing, this will again give us an issue like we had before trying to run the code. 
I will remove the click event references while testing and put them back before merging so they remain consistent for both of us. 
i.e. you will need to remove them also to test your feature. 
Let me know if this makes sense.

![image](https://github.com/user-attachments/assets/2c1f86fc-efbf-4ce4-8256-b57daed13be5)

**Comment by ryanhtafesa** [comment]
@AlexandraKoshur 
I am merging mortgage calculator into development now.

**Reply by AlexandraKoshur** [reply]
@ryanhtafesa 
Good luck, thank you for keeping me updated!

**Comment by ryanhtafesa** [comment]
@AlexandraKoshur 
Merge has been completed. Ready for your to pull the latest development and merge with your branch. 
When you're ready, let me know so we can work through the merge conflicts

**Reply by AlexandraKoshur** [reply]
@ryanhtafesa I will try now and let you know how I go.

**Reply by ryanhtafesa** [reply]
Thank you and good luck!
Feel free to get me on teams when you're ready for the next step

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa 
I just pushed the development branch. Thank you for helping me to resolve the conflict after merging my feature_currency! Let me know how you're doing with the further merging.

**Reply by ryanhtafesa** [reply]
All done from my end - we should be able to proceed with the final pull request task
